src/provider/HttpFile/ConfigurationServiceHttpFile.ts

```diff
@@ -22,16 +22,32 @@ import {
   getRepositoryKeyNotFoundErrorMessage
 } from '../../utils';
 
+interface ConfigurationServiceHttpFileRequest {
+  token: string,
+  disableCache?: boolean
+}
+
 export class ConfigurationServiceHttpFile implements ConfigurationService {
-  constructor(private token: string) {
+  private token: string = '';
+  private disableCache?: boolean;
+
+  constructor(request: string | ConfigurationServiceHttpFileRequest) {
+    if(typeof request === 'string') {
+      this.token = request;
+    }
+    else if(typeof(request) === 'object') {
+      this.token = request.token;
+      this.disableCache = request.disableCache;
```

Comment:
this.disableCache = request.disableCache || false;

Reply:
done